packages/actionmenu/src/react/item.tsx

```diff
@@ -47,15 +52,29 @@ const styles: { [name: string]: StyleFn } = {
   textOnly: () => css(stylesheet['.psds-actionmenu__text-only'])
 }
 
-interface ItemProps extends Omit<HTMLAttributes<HTMLLIElement>, 'onClick'> {
-  className?: string
+interface BaseItemProps {
   disabled?: boolean
-  href?: string
-  nested?: React.ReactNode
-  onClick?: (event: React.MouseEvent, value: React.ReactText) => void
-  origin?: string
-  tagName?: string
-  value?: string | number
+  nested?: ReactNode
+  origin?: ValueOf<typeof origins>
+  value?: ReactText
+  onClick?: (event: MouseEvent, value?: ReactText) => void
+}
+
+interface AnchorProps
+  extends BaseItemProps,
+    Omit<HTMLPropsFor<'a'>, 'onClick'> {
+  tagName?: 'a'
```

Comment:
Why introduce another patten for this and not use the existence of href or the renderProp patterns that we have so far?

Reply:
to avoid a breaking change. this is the pattern that was used in ActionMenu - i just added the typings